vite.config.ts

```diff
@@ -1,5 +1,7 @@
+/// <reference types="vitest/config" />
+// @ts-expect-error could not resolve types for @vitejs/plugin-react
 import react from "@vitejs/plugin-react"
```

Comment:
Complaining about missing type definitions when moduleResolution set to `node`. We can later look into alternative solutions